pkg/server/ca/manager/slot.go

```diff
@@ -53,11 +54,8 @@ func (s *SlotLoader) Load(ctx context.Context) (*Journal, map[SlotPosition]Slot,
 	log := s.Log
 	journalPath := s.journalPath()
 
-	var currentX509CA *X509CASlot
-	var nextX509CA *X509CASlot
-	var currentJWTKey *JwtKeySlot
-	var nextJWTKey *JwtKeySlot
-
+	// TODO: looks like it is old and no longer required
+	// DEPRECATED: remove on 1.7?
```

Comment:
It's probably better to create an issue to look into this rather than having this comment here?